src/views/Predictions/context/SwiperProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { createContext, Dispatch, useState } from 'react'
```

Comment:
seems really generic, maybe move it to `src/context`

Reply:
Generic, but specific to the Predictions. I would rather not wrap the entire app for just the context that is used in the predictions.